src/platform/windows/display.h

```diff
@@ -163,6 +163,8 @@ namespace platf::dxgi {
       return (capture_format == DXGI_FORMAT_R16G16B16A16_FLOAT) ? 8 : 4;
     }
 
+    bool mouse_pointer_visible = false;
```

Comment:
What's the reason for hiding the mouse cursor on Windows?

Reply:
It's not *us* hiding it. If there is no mouse pointer present on specific monitor at specific time, we don't need to blend it and our logic changes slightly.